src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicallyAccessedMembersAttribute.cs

```diff
@@ -26,7 +26,12 @@ namespace System.Diagnostics.CodeAnalysis
         AttributeTargets.Field | AttributeTargets.ReturnValue | AttributeTargets.GenericParameter |
         AttributeTargets.Parameter | AttributeTargets.Property | AttributeTargets.Method,
         Inherited = false)]
-    public sealed class DynamicallyAccessedMembersAttribute : Attribute
+#if SYSTEM_PRIVATE_CORELIB
```

Comment:
I don't think I understand why you need this?

Reply:
I understand why you need it now. I'm not sure I agree with what we are doing here to be honest, but I'm fine if that is the direction we are going to take.

Reply:
Yes, that is the direction we've decided to take. See https://github.com/dotnet/runtime/issues/36656#issuecomment-630345623

The other attributes in this folder already have it:

https://github.com/dotnet/runtime/blob/8126962457be677cb8b9313d5d9f26849da77df2/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/DynamicDependencyAttribute.cs#L17-L22

https://github.com/dotnet/runtime/blob/8126962457be677cb8b9313d5d9f26849da77df2/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs#L9-L14

Reply:
I see, honestly I don't fully agree with this approach (as opposed to having them available down level) since we probably want to allow library developers that target netstandard or < .NET5.0 to be able to annotate their libraries as well, specially if these will be libraries used for blazor wasm. Given the attribute implementation is fully netstandard2.0 compatible, I would opt instead for having compatibility packs (similar to what we did with AsyncInterfaces) for shipping the attributes down level so that we a) don't have to compile these attributes internally on a bunch of libraries and b) allow third party library writters to consume them. I don't want to push to much on doing this but just wanted to share my opinion 😄 cc: @stephentoub